curriculum/js1/10.js

```diff
@@ -7,10 +7,15 @@
  * @param {function} c
  */
 
-const solution = (a, b, c) => {
-};
+ const solution = (a, b, c) => {
+   setTimeout(function(){
+     c();
+     setTimeout(function(){
```

Comment:
c is a function, so its pointless to define a function here to call that function. `setTimeout(c, b)`